platform/darwin/src/MGLRasterStyleLayer.h

```diff
@@ -154,13 +154,6 @@ MGL_EXPORT
  */
 @property (nonatomic, null_resettable) MGLStyleValue<NSNumber *> *rasterFadeDuration;
 
-/**
- The transition affecting any changes to this layer’s `rasterFadeDuration` property.
-
- This property corresponds to the `raster-fade-duration-transition` property in the style JSON file format.
-*/
-@property (nonatomic) MGLTransition rasterFadeDurationTransition;
```

Comment:
Is the removal of this property intentional? It’s is present in iOS map SDK v3.7.0, so typically we’d deprecate it before removing it outright.

Reply:
Yes, see https://github.com/mapbox/mapbox-gl-js/commit/fa472086d4c3027a9a717f7793265272b259a875. I'd advocate for outright removing it rather than deprecating: it's _very_ obscure, and this is for a semver major release anyway.

Reply:
Makes sense. I guess in the future we can also subsume `raster-fade-duration` under a transition property somehow for consistency.